apps/snitch_core/lib/core/data/model/payment/payment.ex

```diff
@@ -62,4 +62,22 @@ defmodule Snitch.Data.Model.Payment do
   def to_subtype(%Payment{payment_type: "ccd"} = payment) do
     CardPaymentModel.from_payment(payment.id)
   end
+
+  defmacro to_char(field, format) do
+    quote do
+      fragment("to_char(?, ?)", unquote(field), unquote(format))
+    end
+  end
```

Comment:
@arjun289 @SagarKarwande shouldn't this become a part of some generic model in the codebase? Thoughts

Reply:
This can be made a part of some database helper as it is used multiple times.